### ISSUE TITLE
Error when pressing Git button in RTC

### ISSUE THREAD

**Issue by spliffone** [jazz-community/rtc-git-commit-picker#3]
When pressing the Git-Button in RTC a message is displayed:

> The repository "xxxxxx" is not a GitLab repository. Currently only GitLab repositories are supported.

Details of the registered Git-Repository...
**Authentication-Method:** E-Mail + Password
**GitLab Version:** GitLab Community Edition 8.1.3

**Comment by innerjoin** [comment]
@spliffone 
We have overhauled the whole plugin and will release a new version in the next five days. It contains various improvements, one of it is a secure usage of Personal Access Tokens to overcome #2. Please note that the current release (1.0.3), which you were trying out, is using *v4* of the GitLab API. The new API endpoint was introduced in GitLab 9.0, so this release and also the one we will be releasing in the next few days will only be **compatible from GitLab 9.0** onward. Consequently, we also decided to remove the authentication based on username/password, as this is *deprecated* in API v4. 
We are very sorrry that this was not clearly stated in the README, we will also add this information to the README with the upcoming release (#1) .

To sum things up, I would suggest to either upgrade to one of the latest GitLab releases (9.x or 10.x) or to help us with a backport for previous releases (i created #4 to track it's implementation).

**Comment by innerjoin** [comment]
@spliffone 
One side note for Siemens employees: You are already using the *new release that will be published on GitHub in five days*. Please comment on our release anouncement on SSN to get in touch with you, so we can discuss about a backport for GitLab 8.1.3

**Comment by spliffone** [comment]
@innerjoin 
Thx for sharing the information, I will contact you using SSN.

**Comment by innerjoin** [comment]
@spliffone 
Thank you for contacting us. 

The code is updated, the issues and limitations outlined, therefore, I will close this issue
